### DISCUSSION TITLE
Joyboy Community Call #1

### DISCUSSION THREAD

**Discussion by AbdelStark** [keep-starknet-strange/joyboy#96] [Announcements]
# Joyboy - 1st Community Call Announcement

Joy Morning.

<a href="https://starkware-co.zoom.us/j/88381437943?pwd=9wXwTewCrrKFF4vbGWMeKIrZE5IVva.1"><img src="https://img.shields.io/badge/Join%20on-Zoom-blue?logo=zoom" height="75" /></a>

## Agenda
- Joy Morning by @AbdelStark (~5min)
- Nostr deep dive by @supertestnet (~20min + 10min of questions)
- Joyboy project overview by @AbdelStark (~15min)
- Backlog review @MSghais (~5min)
- OD Hack summary @MSghais (~5min)
- Building a blockchain mobile consumer app by @ugur-eren (~5-10min)
- Community discussions round (time depending on the vibes and content)

## Information
- When: [Thur, 30 May 2024 at 5:30 PM CEST](https://www.timeanddate.com/worldclock/converter.html?iso=20240530T153000&p1=195&p2=179&p3=1440)

**Comment by rohitgoyal** [comment]
Any recording on this call?

**Reply by ugur-eren** [reply]
https://youtu.be/6ZHD28NZI30
